gensrc/thrift/WorkGroup.thrift

```diff
@@ -0,0 +1,41 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+namespace cpp starrocks
+namespace java com.starrocks.thrift
+
+enum TWorkGroupType {
+  WG_REALTIME,
+  WG_NORMAL,
+  WG_DEFAULT
+}
+
+struct TWorkGroupClassifier {
+  1: optional i64 id
+  2: optional i64 workgroup_id
+  3: optional string user
+  4: optional string role
+  5: optional set<string> query_types
```

Comment:
Why query_type is a set？

Reply:
`create resource_group rg0 to (query_type in ('select', 'insert', 'load')) with (...)` according to product design